.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: android
+jdk: oraclejdk8
 android:
   components:
-    - build-tools-19.1.0
-    - android-19
+    - platform-tools
+    - tools
+    - build-tools-25.0.1
+    - android-25
+node_js:
```

Comment:
did you intend the language set to be node_js instead of android ?
I just sent this as an initial try.